osf/management/commands/update_mailchimp_email.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import datetime
+
+from django.core.management.base import BaseCommand
+from osf.models import OSFUser
+from website import mailchimp_utils
+
+logger = logging.getLogger(__name__)
+
+
+def update_mailchimp_email():
+    users_updated = 0
+    for user in OSFUser.objects.filter(deleted=None):
```

Comment:
nit: what's the difference between this and `deleted__is_null=True`, which is mostly what I've seen elsewhere?

Reply:
I dont **think** there's a difference, and I'm not finding any django literature that says theres a difference. I will change it to deleted_isnull=True, as it is more common.